src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+  const [panelOne, setPanelOne] = useState("0");
+  const [panelTwo, setPanelTwo] = useState("0");
+  const [panelMath, setPanelMath] = useState("+");
+  const [answer, setAnswer] = useState(0);
+
+  const addToPanelOne = (event) => {
```

Comment:
your function names can improve to be a bit cleare: `updateLeftNumber` for example